src/oic/utils/http_util.py

```diff
@@ -225,12 +229,16 @@ def extract(environ, empty=False, err=False):
     :param empty: Stops on empty fields (default: Fault)
     :param err: Stops on errors in fields (default: Fault)
     """
+    # FIXME: This seems to be unused -> Deprecate?
```

Comment:
Thats basic wsgi parsing, so yes, deprecate if not in use.

Reply:
Done.